all/remote/pom.xml

```diff
@@ -332,6 +347,35 @@
                </execution>
             </executions>
          </plugin>
+         <plugin>
+            <artifactId>maven-source-plugin</artifactId>
+            <version>2.3</version>
+            <executions>
+               <execution>
+                  <id>attach-sources</id>
+                  <phase>none</phase>
+               </execution>
+            </executions>
+         </plugin>
+         <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>2.4</version>
+            <executions>
+               <execution>
+                  <id>build-test-jar</id>
+                  <phase>none</phase>
+               </execution>
+            </executions>
+         </plugin>
+         <plugin>
+            <groupId>org.apache.karaf.tooling</groupId>
+            <artifactId>karaf-maven-plugin</artifactId>
+            <configuration>
+               <features>
+                  <feature>infinispan-remote</feature>
+               </features>
+            </configuration>
+         </plugin>
```

Comment:
I can't comment on this not knowing much about how you build modules. For what I can see, you are not creating features files in `all/embedded-query` so probably not.